algorithmic_efficiency/workloads/criteo1tb/input_pipeline.py

```diff
@@ -39,13 +39,15 @@ def _parse_example_fn(example):
 
     int_features = []
     for idx in range(num_dense):
-      int_features.append(fields[idx + num_labels])
+      int_features.append(tf.math.log(fields[idx + num_labels] + 1))
     int_features = tf.stack(int_features, axis=1)
 
     cat_features = []
     for idx in range(len(vocab_sizes)):
       cat_features.append(
-          tf.cast(fields[idx + num_dense + num_labels], dtype=tf.int32))
+          tf.io.decode_raw(fields[idx + num_dense + num_labels], tf.int64)[:, 0]
+          % vocab_sizes[idx])
+    print(cat_features)
```

Comment:
Do you want to keep the `print` statement here or was this just for debugging?

Reply:
ah thanks for catching that, removed.